test/ziggurat/kafka_consumer/consumer_test.clj

```diff
@@ -1,33 +1,20 @@
 (ns ziggurat.kafka-consumer.consumer-test
-  (:require [clojure.test :refer :all])
-  (:require [ziggurat.kafka-consumer.consumer :refer [create-consumer]]
-            [ziggurat.config :refer [ziggurat-config]]
-            [ziggurat.fixtures :as fix])
-  (:import (org.apache.kafka.clients.consumer KafkaConsumer)
-           (java.util Properties)))
+  (:require [clojure.test :refer [deftest is testing use-fixtures]])
+  (:require [ziggurat.config :refer [ziggurat-config]]
+            [ziggurat.fixtures :as fix]
+            [ziggurat.kafka-consumer.consumer :refer [create-consumer]])
+  (:import (org.apache.kafka.clients.consumer KafkaConsumer)))
 
 (use-fixtures :once fix/mount-only-config)
 
```

Comment:
Each namespace should have one test for backward compatibility - where we can check if older configs for that namespace are set properly.

And, one test for non-kafka-config-keys

Reply:
same as https://github.com/gojek/ziggurat/pull/219#discussion_r614615370

Reply:
I think going forward its better to have the ziggurat configs and kafka configs separate instead all of them on a single map...